lib/transform.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs').promises;
+
+const transformFile = (src) => {
+  const promiseFile = fs.readFile(src, { encoding: 'utf-8' });
+  return promiseFile
+    .then(contents => contents.replace(/[^A-Z]/g, ''))
```

Comment:
`/[^A-Z]/g` means to replace anything that is not capitalized. 

You would want to use `/[A-Z]/g`